src/p2a_analyses_read_processed_data.R

```diff
@@ -107,25 +107,30 @@ data <- data[-which(data$name == "Ben" & data$dt == as.POSIXct(strptime("2016-11
 #### STEP 4: ANNOTATE GEOGRAPHICAL INFORMATION
 ################################################
 library(rgdal);library(sp);library(raster)
+
+download.file("https://www.naturalearthdata.com/http//www.naturalearthdata.com/download/50m/cultural/ne_50m_admin_0_countries.zip","data/maps/ne_50m_admin_0_countries.zip")
```

Comment:
I would add an `if` statement before the download to check if the file already exists. If it does not exists, download, unzip,... If already there is should not be re-downloaded. Something like:

```
if (!file.exists(...)) {
# download the file, unzip,...
}
```

`
`

Reply:
Good point! I included an if statement to avoid re-downloading files.